src/main/java/dev/walshy/sfmetrics/charts/MetricVersionChart.java

```diff
@@ -0,0 +1,19 @@
+package dev.walshy.sfmetrics.charts;
+
+import dev.walshy.sfmetrics.MetricsModule;
+import org.bstats.bukkit.Metrics;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MetricVersionChart extends Metrics.AdvancedPie {
```

Comment:
Why an AdvancedPie?
It's not like someone can be running multiple metrics versions. A simple pie should be better.

Reply:
Thought I had set tha treee